.gitignore

```diff
@@ -5,6 +5,7 @@ dist
 build
 docs/build
 docs/source/reference/services
+src/
```

Comment:
Why did this get added?

Reply:
@stealthycoin Great question. I'm not the original committer, so I can't say for sure. But it looks like a mistake/oversight to me. I'm happy to remove it and update the PR if you want me to.

Reply:
I updated the PR to remove this change.